azure_monitor/examples/client.py

```diff
@@ -1,22 +1,8 @@
-# Copyright 2019, OpenCensus Authors
```

Comment:
We need Microsoft copyright and license information, something small like this will be fine I suppose

# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License.